app/views/layouts/basic.html.erb

```diff
@@ -0,0 +1,58 @@
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width, initial-scale=1" name="viewport">
+    <title> 
+      <% if content_for(:title) %>
+        <%= yield(:title) %>
+      <% elsif @cms_page %>
+        <%= @cms_page.label %>
+      <% else %>
+        Library
+      <% end %>
+    </title>
+    <link href="https://www.ualberta.ca/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180">
+    <link color="#275d38" href="https://www.ualberta.ca/safari-pinned-tab.svg" rel="mask-icon">
+    <link href="https://www.ualberta.ca/favicon.svg" rel="icon">
+    <link href="https://www.ualberta.ca/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png">
+    <link href="https://www.ualberta.ca/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png">
+    <script crossorigin="anonymous" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
```

Comment:
https://github.com/ualbertalib/library-cms/blob/fec21bc784055fdd20c1c70e3404fc71ac0cdb53/app/views/layouts/basic.html.erb#L23 pulls Bootstrap v5.2.3 from line 23 of https://www.ualberta.ca/_assets/css/framework-v2.css

I wonder if that alone would suffice and then this would not be needed.